### PR TITLE
reworked usage/description for netclient and added verbosity flags

### PR DESCRIPTION
**Pull request by 0xdcarns**

### CODE REVIEW

File: netclient/cli_options/flags.go [hunk 1]
```diff
@@ -204,5 +204,23 @@ func GetFlags(hostname string) []cli.Flag {
 			Value:   "no",
 			Usage:   "Allows to run the command with force, if otherwise prevented.",
 		},
+		&cli.BoolFlag{
+			Name:    "verbosity-level-1",
+			Aliases: []string{"V"},
```
Comment (mattkasun) [inline netclient/cli_options/flags.go:209]: upper case?  isn't lower case more common.

File: netclient/config/config.go [hunk 2]
```diff
@@ -25,7 +25,6 @@ type ClientConfig struct {
 	Daemon          string         `yaml:"daemon"`
 	OperatingSystem string         `yaml:"operatingsystem"`
 	DebugOn         bool           `yaml:"debugon"`
```
Comment (mattkasun) [inline netclient/config/config.go:27]: This var can be deleted and rework where it is used to log by verbosity to achieve the same functionality

**Review by mattkasun — COMMENTED** [review]
couple of questions.  Changes not necessarily required.

File: netclient/cli_options/flags.go (hunk 1 above)
Comment (0xdcarns) [reply netclient/cli_options/flags.go:209]: yeah you're right, I'll change to lowercase, I was running into issues with lowercase at some point and left at uppercase for some reason

**Pull request merged**